service/controller/cluster.go

```diff
@@ -93,7 +92,7 @@ func newClusterResources(config ClusterConfig) ([]resource.Interface, error) {
 	{
 		c := app.Config{
 			ChartName:      config.ChartName,
-			G8sClient:      config.K8sClient.G8sClient(),
+			G8sClient:      config.K8sClient.CtrlClient(),
```

Comment:
Keeping the G8sClient name makes sense to me 👍 

I've been trying to update it but we're still using the generated client in a lot of places in app platform. I think we can remove it at the same time as switching to apiextentions-application.

Reply:
That was actually an oversight on my part, but it's something we can change over time.